viewer/src/assets/schema.json

```diff
@@ -1,149 +1,149 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "Wallet",
-  "title": "Wallet",
+  "$id": "CredentialAgent",
```

Comment:
Can we agree that CredentialAgent is better than Wallet and replace that with a better term if we come up with one?

I wouldn't mind if we just had agents with issuer, holder and verifier capabilities, and a special nickname wallet for holder agents.

In some contexts, it may be worthwhile to separate holder agent and wallet from each other. For example the [ToIP Glossary](https://docs.google.com/document/d/1fZByfuSOwszDRkE7ARQLeElSYmVznoOyJK4sxRvJpyM/edit#heading=h.z3ugzt4hgdf6) specifies: "an [agent](https://docs.google.com/document/d/1fZByfuSOwszDRkE7ARQLeElSYmVznoOyJK4sxRvJpyM/edit#heading=h.6xkhfkjpo6xg) (specifically a type of [software agent](https://docs.google.com/document/d/1fZByfuSOwszDRkE7ARQLeElSYmVznoOyJK4sxRvJpyM/edit#heading=h.qv9e403ohhxe)) that operates in conjunction with a [digital wallet](https://docs.google.com/document/d/1fZByfuSOwszDRkE7ARQLeElSYmVznoOyJK4sxRvJpyM/edit#heading=h.sxnvf3f5v156)"

IMHO we don't need that complexity in this overview. Or can people choose a separate holder wallet and a holder agent from the overview and make them work in conjunction?